cmd/snap/cmd_changes.go

```diff
@@ -51,15 +51,18 @@ type cmdChange struct {
 }
 
 type cmdTasks struct {
-	Positional struct {
-		ID changeID `positional-arg-name:"<id>" required:"yes"`
+	LastChangeType string `long:"last"`
+	Positional     struct {
+		ID changeID `positional-arg-name:"<id>"`
 	} `positional-args:"yes"`
 }
 
 func init() {
 	addCommand("changes", shortChangesHelp, longChangesHelp, func() flags.Commander { return &cmdChanges{} }, nil, nil)
 	addCommand("change", shortChangeHelp, longChangeHelp, func() flags.Commander { return &cmdChange{} }, nil, nil).hidden = true
-	addCommand("tasks", shortChangeHelp, longChangeHelp, func() flags.Commander { return &cmdTasks{} }, nil, nil)
+	addCommand("tasks", shortChangeHelp, longChangeHelp, func() flags.Commander { return &cmdTasks{} }, map[string]string{
+		"last": "Show last change of given type (install, refresh, remove etc.)",
```

Comment:
we should list "auto-refresh" here as well, it's an interesting one and not obviously a command